### PR TITLE
Training Updates & VQA documentation updates

### PR DESCRIPTION
**Pull request by finalelement**
Readme for Training Recipe of M3
Training recipes 
VQA screenshots & Readme updates 
Dataset_mixture updates 
Bash scripts

### CODE REVIEW

File: m3/data_prepare/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # MONAI-VILA: Data Preparation
 
+Preparing the datasets for VILA training and testing requires three steps:
+1. Downloading all the datasets (Information to download each dataset is provided in the readme.md for the `vqa`, `report` and `expert` directories)
+2. Generating the instruction data for all datasets (Information to generate the instruction data is provided in the readme.md for the `vqa`, `report` and `expert` directory)
+3. Adding the prepared datasets to VILA in a data mixture (More information can be found in the quickstart guide))
```
Comment (holgerroth) [inline m3/data_prepare/README.md:6]: can you make quickstart guide a link?

**Comment by holgerroth** [comment]
The images should be placed under docs here https://github.com/Project-MONAI/VLM/tree/main/m3/docs/images

File: m3/train/m3-vila8b.sh [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+echo "MASTER_ADDR="$MASTER_ADDR
+CURRENT_RANK=${SLURM_PROCID:-"0"}
+n_node=${SLURM_JOB_NUM_NODES:-1}
+echo "JobID: $SLURM_JOB_ID | Full list: $worker_list | rank $CURRENT_RANK of $n_node"
+
+bs=16
+STAGE2_PATH=$STAGE2_PATH
+OUTPUT=provide_output_dir_name
+
+HEALTHCARE_DS=$(for i in {1..10}; do echo -n usmle+; done)
```
Comment (holgerroth) [inline m3/train/m3-vila8b.sh:43]: Can you add a comment here in the scripts to describe what this is doing, e.g. "# UPSAMPLING FOR DATA BALANCING"

**Review by holgerroth — COMMENTED** [review]
Looks good but left some minor comments

**Comment by finalelement** [comment]
@holgerroth all addressed.

**Review by holgerroth — APPROVED** [review]
Nice job!

**Pull request merged**